SPECS/blktrace/blktrace.spec

```diff
@@ -1,15 +1,13 @@
-%define sha1 blktrace=22a258ea65c6e826596b8e5a51e9c3f8bf758752
 Summary:        Utilities for block layer IO tracing
 Name:           blktrace
-Version:        1.2.0
-Release:        7%{?dist}
+Version:        1.3.0
+Release:        1%{?dist}
 License:        GPLv2
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Development/Tools/Other
 URL:            https://git.kernel.org/cgit/linux/kernel/git/axboe/blktrace.git/tree/README
-Source0:        https://git.kernel.org/pub/scm/linux/kernel/git/axboe/blktrace.git/snapshot/%{name}-%{version}.tar.gz
-Patch0:         blktrace-fix-CVE-2018-10689.patch
+Source0:        https://brick.kernel.dk/snaps/%{name}-%{version}.tar.gz
```

Comment:
I believe the Source0 for this should be on kernel.org.  Block Trace was added to the kernel quite some time ago
https://git.kernel.org/pub/scm/linux/kernel/git/axboe/blktrace.git/snapshot/blktrace-1.3.0.tar.gz